install/changelog.md

```diff
@@ -52,6 +57,86 @@ title: "Change Log"
 
 > See [known issues](../reference/known-issues.html) for currently known issues and workarounds.
 
+## v0.15.0 {#v150}
+
+Released on August 13, 2018
+
+In addition to the 0.15.0 CLI release, we've released new versions of all of our packages.
+
+### Pulumi CLI
```

Comment:
We should also include major features in the service.

At the very least, this should include our GitHub App. /cc @chrsmith as he agreed to write this (I suggest we all just push to this PR).